go/vt/vtorc/inst/instance_dao.go

```diff
@@ -1113,11 +1104,11 @@ func ForgetLongUnseenInstances() error {
 func SnapshotTopologies() error {
 	writeFunc := func() error {
 		_, err := db.ExecVTOrc(`
-        	insert ignore into
+        	insert or ignore into
         		database_instance_topology_history (snapshot_unix_timestamp,
         			alias, hostname, port, source_host, source_port, keyspace, shard, version)
         	select
-        		UNIX_TIMESTAMP(NOW()),
+        		strftime('%%s', 'now'),
```

Comment:
@dbussink I think this needs to be `%s`
```suggestion
        		strftime('%s', 'now'),
```

Reply:
Good catch, applied!